src/tests/unit/doughs-dao.test.js

```diff
@@ -0,0 +1,85 @@
+import chai from 'chai';
+import chaiAsPromised from 'chai-as-promised';
+import sinon from 'sinon';
+
+import { getDoughs } from 'api/v1/db/oracledb/doughs-dao';
+import * as doughsSerializer from 'api/v1/serializers/doughs-serializer';
+import * as connectionApi from 'api/v1/db/oracledb/connection';
```

Comment:
This should be imported before `import { getDoughs } from 'api/v1/db/oracledb/doughs-dao';`

> Each line within groups 1-3 should be ordered alphabetically by the first alphanumeric character in the name of the package/file in the statement.

Ref: https://github.com/osu-mist/style-guides/blob/master/JavaScript.md#importing-modules